pandas/core/indexes/range.py

```diff
@@ -313,6 +313,20 @@ def stop(self) -> int:
         """
         The value of the `stop` parameter.
 
+        This property returns the `stop` value of the RangeIndex, which defines the
```

Comment:
Could we make this just 2 sentences? I think the information describing the RangeIndex is superfluous.

Reply:
done!